src/main/java/seedu/address/logic/commands/contactcommands/SortContactCommand.java

```diff
@@ -0,0 +1,23 @@
+package seedu.address.logic.commands.contactcommands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.model.Model;
+import seedu.address.model.person.PersonNameComparator;
+
+public class SortContactCommand extends Command {
```

Comment:
previously raised, but maybe it should be SortCommand? or we change everything to fit e.g. AddContact...